### PR TITLE
ROC-2506: Document management integration

### PR DESCRIPTION
**Pull request by rlewan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/ROC-2506

### Change description ###

Document management integration.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -285,3 +288,70 @@ services:
...
+        - ROOT_APPENDER
+        - REFORM_SERVICE_TYPE=node
+        - REFORM_SERVICE_NAME=document-management-api-gateway-web
+        - REFORM_TEAM=cc
+        - REFORM_ENVIRONMENT=docker
+        - XFWD=true
+      mem_limit: 256m
+      memswap_limit: 0
+      ports:
+        - 8085:8080
```
Comment (geek-soft) [inline docker-compose.yml]: Do not expose ports in main docker-compose.yml file.

File: docker-compose.yml [hunk 2]
```diff
@@ -285,3 +288,70 @@ services:
...
+          - ROOT_APPENDER
+          - REFORM_SERVICE_TYPE=java
+          - REFORM_SERVICE_NAME=document-management-api
+          - REFORM_TEAM=cc
+          - REFORM_ENVIRONMENT=docker
+          - LOGGING_LEVEL_ROOT=DEBUG
+       mem_limit: 512m
+       memswap_limit: 0
+       ports:
+         - 8084:8080
```
Comment (geek-soft) [inline docker-compose.yml]: The same here.

File: docker-compose.yml (hunk 1 above)
Comment (dharmendrak) [reply docker-compose.yml]: removed

File: docker-compose.yml (hunk 2 above)
Comment (dharmendrak) [reply docker-compose.yml]: removed

File: docker-compose.yml [hunk 3]
```diff
@@ -285,3 +288,66 @@ services:
       image: mailhog/mailhog
       mem_limit: 32m
       memswap_limit: 0
+    document-management-api-gateway-web:
+      image: docker.artifactory.reform.hmcts.net/evidence/em-api-gateway-web:latest
```
Comment (geek-soft) [inline docker-compose.yml]: Please do not use latest. Our experience proves that third party dependencies are unstable. Please fix it to latest version which suits your needs.

File: docker-compose.yml [hunk 4]
```diff
@@ -285,3 +288,66 @@ services:
...
+      memswap_limit: 0
+      depends_on:
+       document-management-api:
+         condition: service_started
+       idam-api:
+         condition: service_started
+       service-auth-provider-api:
+         condition: service_started
+    document-management-api:
+       image: docker.artifactory.reform.hmcts.net/evidence/evidence-management-store:latest
```
Comment (geek-soft) [inline docker-compose.yml]: The same here.

**Pull request merged**